pandas/tests/groupby/test_value_counts.py

```diff
@@ -79,3 +79,25 @@ def rebuild_index(df):
     # have to sort on index because of unstable sort on values
     left, right = map(rebuild_index, (left, right))  # xref GH9212
     tm.assert_series_equal(left.sort_index(), right.sort_index())
+
+
+@pytest.mark.parametrize("freq", ["1D", "2D", "1W", "1Y"])
```

Comment:
can you add the original issue as a self contained test.

Reply:
what is a self-contained test?
does it mean that the separated module file? or a test like this?:
```python
def test_series_groupby_value_counts_with_grouper():
    # GH28479
    df = pd.DataFrame({'Timestamp':[1565083561, 1565083561+86400, 1565083561+86500, 1565083561+86400*2, 1565083561+86400*3, 1565083561+86500*3, 1565083561+86400*4],
                       'Food':['apple', 'apple', 'banana', 'banana', 'orange', 'orange', 'pear']}).drop([3])
    
    df['Datetime'] = pd.to_datetime(df['Timestamp'].apply(lambda t:str(t)), unit='s')
    dfg = df.groupby(pd.Grouper(freq='1D', key='Datetime'))

    # have to sort on index because of unstable sort on values xref GH9212
    result = dfg['Food'].value_counts().sort_index()
    expected = dfg['Food'].apply(Series.value_counts).sort_index()
    expected.index.names = result.index.names

    tm.assert_series_equal(result, expected)
```

Reply:
i want an exact reproduction of the original issue (in the same test file)

Reply:
ok, I'll change so.

Reply:
It had been added.